### DISCUSSION TITLE
Whoops it looks like you got a 415?

### DISCUSSION THREAD

**Discussion by Karinalombera** [deepgram/community#851] [General help]
Hi,
When I try to translate a voice to a text a massage appears. "whoops it looks like you got a 415". And I'm not sure how to solve it

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jjmaldonis** [comment]
Hi @Karinalombera, a `415` error is an `Unsupported Media Type` [error](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/415). What type of audio file are you sending to Deepgram?

**Comment by Karinalombera** [comment]
In one of the Mac apps, voice memos. I have my recording, accordingly the
file is M4A. furthermore my audio longs 1hr 36 min.

**Reply by jjmaldonis** [reply]
M4A files should be supported. Can you record a short voice memo, check if it returns a `415` (you can use the [playground](https://playground.deepgram.com/) to see if the request fails), and upload the audio file here if the request fails?

We can take a look at that specific audio file.

**Comment by Karinalombera** [comment]
When I select the voice memos app, do not allow me to select a specific
audio from the app. It looks like the deepgram transcript all the audios I
have on the Mac app.

**Comment by Karinalombera** [comment]
I did select API playground
